CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Document new compile-time dependency `mercurial` in user-facing documentation. ([#1683](https://github.com/operator-framework/operator-sdk/pull/1683))
 - Adds new flag `--zap-time-encoding` to the flagset provided by `pkg/log/zap`. This flag configures the timestamp format produced by the zap logger. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) for more information. ([#1529](https://github.com/operator-framework/operator-sdk/pull/1529))
+- Improve error message regards dependency manager `modules` requires env var GO111MODULE=on ([#1776](https://github.com/operator-framework/operator-sdk/pull/1776))
```

Comment:
Probably better to leave this out of the CHANGELOG.
The CHANGELOG is more suited for user facing API/CLI/behavioral/workflow changes.

Reply:
👍